config/flags_terms.go

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (C) 2020 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package config
+
+import "github.com/urfave/cli/v2"
+
+var (
+	// FlagAgreedTermsConditions agree with terms & conditions.
+	FlagAgreedTermsConditions = cli.BoolFlag{
+		Name:  "agreed-terms-and-conditions",
+		Usage: "Agree with terms & conditions for consumer, provider or both depending on the command executed",
+	}
+
+	// FlagTermsConsumerAgree marks that consumer TOS has been agreed.
+	FlagTermsConsumerAgree = cli.BoolFlag{
```

Comment:
I don't understand why we need multiple flags.

Reply:
Tequilapi terms API can have more granular flags, but cli command should be more user friendly and have only `--accepted-terms-of-service`

Reply:
@Waldz  I made these flags `hidden` to make it more obvious that the user doesn't have to pass them.

I basically described them because of a single reason, it's easier to read the config if they're described and set.
User only passes the `--accepted-terms-of-service` and we update these flags for him.
While he does have the option to pass these flags himself, he does not have to.

This is done for the reasons stated in the PR description: so that we can parse the config easily, know which flags to check and not have to ask tequila anything. This makes it easier to control and check TOS because we dont have to request them from the API. Sometimes we have to check before the API is even started which makes things even worse.

Another way to do this, would be to define custom config consts somewhere and store them, parsing the config in that regard, though I feel like that would be even more confusing and obscure.